infrastructure/README.md

```diff
@@ -52,3 +59,30 @@ This folder contains all the Terraform files for provisioning infrastructure in
    products-static-web-url = https://mhraproductsnonprod.z33.web.core.windows.net/
    search_admin_key = CB28B1A47E29FF4620184BD27B89945E
    ```
+
+## Setup a service principal account
+
+1. [Authenticate](https://www.terraform.io/docs/providers/azurerm/guides/azure_cli.html) with Azure
+
+   ```shell
+   az login
+
+   # SUBSCRIPTION_ID can be found in output from previous command
+   az account set --subscription="SUBSCRIPTION_ID"
+   ```
+
+2. [Create a service principal](https://docs.microsoft.com/en-us/cli/azure/create-an-azure-service-principal-azure-cli?view=azure-cli-latest#password-based-authentication) password based authentication
```

Comment:
Should this describe what `<ServicePrincipalName>` should be?

Reply:
I add a coment to replace `<ServicePrincipalName>` with the name of the account you want to create